model/src/cfd.rs

```diff
@@ -1737,13 +1815,13 @@ pub enum Completed<P, E> {
     },
 }
 
-impl<P, E> xtra::Message for Completed<P, E>
```

Comment:
I think we may be able to get rid of this enum if we instead have separate commands for these:

- `setup_contract_failed`
- etc

That was the original plan at least. Certainly, we should no longer be sending this as a message so this can probably go away. Would be nice to do these cleanups in a patch at the front of the PR:

- git reset --soft HEAD^ to remove the commit but keep the changes
- git stash
- make the small patch
- git stash pop